test/app/renderer/api/generateAddress.spec.ts

```diff
@@ -1,26 +1,38 @@
-import { asObject } from "app/common/runtimeTypes"
+import { asObject, asRuntimeType } from "app/common/runtimeTypes"
 import { ipcRendererFactory } from "test/__stubs__/ipcRenderer"
 import { jsonSerializer } from "test/__stubs__/serializers"
 import * as api from "app/renderer/api"
-//import { routes } from "app/main/api"
+import { routes } from "app/main/api"
 import { GenerateAddressParams, GenerateAddressSuccess } from "app/common/runtimeTypes/ipc/address"
 
 describe("GenerateAddress API", () => {
   // Params
-  const generateAddressParams: GenerateAddressParams = {
+  const generateAddressParams: GenerateAddressParams = asRuntimeType({
     account: 0,
     label: "From Satoshi Nakamoto",
-    amount: 1,
+    requestedAmount: 1,
     expirationDate: 1526342400  // May 15, 2018
-  }
+  }, GenerateAddressParams)
 
   // Response
-  const generateAddressSuccess: GenerateAddressSuccess = {
+  const generateAddressSuccess: GenerateAddressSuccess = asRuntimeType({
     kind: "SUCCESS",
-    keyPath: [2147483651, 2147488567, 2147483648, 0, 0],
-    address: "wit1qre9fq64r5jgv0t3m8q3tvnqwphxm9xpacvrdhe5",
-    creationDate: 1525478400    // May 5, 2018
-  }
+    finalKey: {
+      kind: "external",
+      extendedKey: {
+        chainCode: "f55975c2fda883d73495932af3974762003dfd715505ea262b1fa3105e157e04",
+        key: "03c0f6ca4b6e580687b955f49705479e0e59e0072db58abdc465c7628582507d54",
+        type: "public"
```

Comment:
not that it really matters but the real `generateAddress` handler would return private external final keys